crates/vm/levm/src/vm.rs

```diff
@@ -346,32 +349,93 @@ impl VM {
         matches!(self.tx_kind, TxKind::Create)
     }
 
-    fn revert_create(&mut self) -> Result<(), VMError> {
-        // Note: currently working with copies
-        let call_frame = self
-            .call_frames
-            .last()
-            .ok_or(VMError::Internal(
-                InternalError::CouldNotAccessLastCallframe,
-            ))?
-            .clone();
+    fn add_intrinsic_gas(&mut self, initial_call_frame: &mut CallFrame) -> Result<(), VMError> {
```

Comment:
Same nit of `thiserror` advantage.

Reply:
https://github.com/lambdaclass/ethrex/pull/1345/commits/92e4848a29c731291131fc3b9b3fb353bd38b369